src/pages/Admin/AdminVenuePreview.tsx

```diff
@@ -93,6 +97,21 @@ export const AdminVenuePreview: React.FC<AdminVenuePreviewProps> = ({
 
   const venueTypeText = infoTextByVenue[venue.template] ?? "Experience Info:";
 
+  const [validBannerImageUrl] = useValidImage(
+    venue.config?.landingPageConfig.bannerImageUrl ??
+      venue.config?.landingPageConfig.coverImageUrl,
+    DEFAULT_VENUE_BANNER
+  );
+
+  const imageVars = useCss({
+    background:
+      validBannerImageUrl === DEFAULT_VENUE_BANNER
+        ? validBannerImageUrl
+        : `url(${validBannerImageUrl})`,
+  });
```

Comment:
I noticed that this piece of code is repeated in the project. Is it possible to move it to a separate function/hook?

Reply:
removed this logic in 208fc95